### DISCUSSION TITLE
Oscilloscope LA104

### DISCUSSION THREAD

**Discussion by nieldk** [gabonator/LA104#53] [Ideas]
It would be cool if oscilloscope app could be made for the LA104.

**Comment by erik-smit** [comment]
The LA104 does not have the hardware to support analog sampling.

**Reply by nieldk** [reply]
Perhaps with a module, for example like this
https://store.digilentinc.com/pmod-ad1-two-12-bit-a-d-inputs/

**Reply by gabonator** [reply]
There already is an application imitating simple oscilloscope. It uses pcf8591t I2C DA converer, you will find it in Devices folder. It can be very easily modified to use any SPI/I2C based DA converter. 

![image](https://user-images.githubusercontent.com/1020041/111881202-baba8480-89af-11eb-8677-cb2328f9e193.png)

![image](https://user-images.githubusercontent.com/1020041/111881252-ee95aa00-89af-11eb-8c50-ecc0b1ec538b.png)
